python/ray/dashboard/modules/metrics/dashboards/serve_deployment_dashboard_panels.py

```diff
@@ -42,7 +42,7 @@
         targets=[
             Target(
                 expr='sum(rate(ray_serve_deployment_error_counter_total{{route=~"$Route",route!~"/-/.*",{global_filters}}}[5m])) by (application, deployment, replica)',
-                legend="{{replica}}",
+                legend="__auto",
```

Comment:
```suggestion
                legend="{{application}}#{{deployment}}#{{replica}}",
```

Can we format it as string instead?

Reply:
Sure!